### ISSUE TITLE
[Accessibility]A11y_ Java Tooling for IntelliJ/Eclipse – New Feature_Storage Container_Non-Text Contrast: In Dark theme, Non-text controls (icons such as delete, refresh, search etc.,) are not visible properly

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/azure-tools-for-java#3560]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yWCAG2.1; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse–NewFeature; #MAS1.4.11; #C+E-H; #JavatoolingforIntelliJ/Eclipse

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS1.4.11

**Note: Similar issue is found in** 
**3.	Spark Submission and** 
**4.	Spark Submission Tab**

**Repro Steps:** 
1) Open Java EE - Eclipse application in Dark theme (WindowsPreferencesAppearanceDark theme)
2) Install 'Azure Toolkit for Eclipse' from Marketplace
3) Open Eclipse and install the Scala plug-in 
4) Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription
5) Tab till Windows menu option-->Show View-->Other-->Expand Azure-->Azure Explorer pane press OK
6) In Azure Explorer section, tab till HDInsight and expand it. Expand the cluster name
7) Tab till the storage container and press enter
8) In the Storage container view, verify if the Non-text controls (icons such as delete, refresh, search etc.,) are visible properly

**Actual Result:** 
In Dark theme, Non-text controls (icons such as delete, refresh, search etc.,) are not visible properly

**Expected Result:** 
In Dark theme, Non-text controls (icons such as delete, refresh, search etc.,) must be visible clearly

**User Impact:** 
Users who are using Dark theme to navigate around the application will not be able to access the controls if the Non-text controls are not visible properly

**MAS Reference:** 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7B6CA3BF14-2635-434A-832C-28DD9A18B9FC%7D&file=MAS%201.4.11%20%E2%80%93%20Non-text%20Contrast.docx&action=default&mobileredirect=true

![Bug_6_In Dark theme, Non-text controls  are not visible properly](https://user-images.githubusercontent.com/41109135/65322009-5274c200-dbc3-11e9-8314-3da84a683ee4.png)
![Bug_6_In Dark theme, Non-text controls (icons such as delete, refresh, search etc ,) are not visible properly](https://user-images.githubusercontent.com/41109135/65322010-5274c200-dbc3-11e9-8bda-209a123949ab.png)

**Comment by wezhang** [comment]
minimum 3:1 contrast ratio

**Comment by Flanker32** [comment]
@Tanushagottipati We have removed storage explorer in IDE toolkit, could you please help us verify and close the issue? Here is the [build](https://dev.azure.com/mseng/VSJava/_build/results?buildId=11079936)

**Comment by Tanushagottipati** [comment]
As mentioned storage accounts tab cannot be opened in the eclipse toolkit. As storage explorer has been deprecated, closing this bug and attached screenshot for reference.

For Spark Submission , refresh control has been removed and For Spark Submission Tab text is written beside the non text controls. hence this issue is also fixed for both of these scenarios

Environment:
Eclipse IDE for Enterprise Java Developers.
Version: 2019-09 R (4.13.0)
Build id: 20190917-1200
Solution Installed: The Azure Toolkit for Eclipse 3.20.0
![Screenshot (709)](https://user-images.githubusercontent.com/41109135/70887207-4dc57b80-2003-11ea-88b2-a6e4081e1ea8.png)
![Screenshot (710)](https://user-images.githubusercontent.com/41109135/70887250-6c2b7700-2003-11ea-887c-c18b6ea6c00c.png)
![Screenshot (711)](https://user-images.githubusercontent.com/41109135/70887254-6fbefe00-2003-11ea-869d-a8f1d9619fd3.png)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749672
